var/spack/repos/builtin/packages/stacktrace/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Stacktrace(CMakePackage):
+    homepage = "https://github.com/AdvancedMultiPhysics/StackTrace"
+    git = "https://github.com/AdvancedMultiPhysics/StackTrace.git"
+
+    maintainers("bobby-philip", "gllongo", "rbberger")
+
+    version("master", branch="master")
```

Comment:
Are there specific tagged versions you can add, maybe by a dated version? Note that if there is a plan to release versions later, it may be useful to label dated versions with a prefixed `0.0` e.g. `0.0.2024-09-01`, to ensure that ordering is correct when 0.1 is released.

Reply:
Added a `0.0.93` version.